app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeController.scala

```diff
@@ -169,7 +175,7 @@ class RepresenteeController @Inject() (
   def enterId(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
     withCapacitorOrPersonalRepresentativeAnswers(request) { (_, journey, answers) =>
       val backLink = answers.fold(
-        _ => routes.RepresenteeController.enterName(),
+        _ => routes.RepresenteeController.enterDateOfDeath(),
```

Comment:
this depends on the representative type. Before `journey` and `answers` above, there is a parameter which tells you the type of representative

Reply:
this still depends on the representative type, it should be something like:
```scala
representativeType.fold(_ => enterDateOfDeath(), _ => enterName())
```
same goes for the back link in the submit endpoint